cmd/provider.go

```diff
@@ -32,6 +32,8 @@ func makeProviderCmd() *cobra.Command {
 
 	command.Flags().String("pull-policy", "Always", `Set to "Always" to force a pull of images upon deployment, or "IfNotPresent" to try to use a cached image.`)
 
+	// command.Flags().Bool("--insecure-registry", false, `Set to true to allow to pull images from a non HTTPS image registry.`)
```

Comment:
That's not far off, but we don't need to include the `--` within the flag syntax.

K3sup has some examples of using the flags package for bools: https://github.com/alexellis/k3sup/blob/master/cmd/join.go

Reply:
I'll definetly check that, thanks for the resource!